scripts/install

```diff
@@ -23,118 +22,151 @@ ynh_abort_if_errors
+ynh_script_progression --message="Creating a MySQL database..." --weight=1
 
+db_name=$(ynh_sanitize_dbid --db_name=$app)
+db_user=$db_name
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_mysql_setup_db --db_user=$db_user --db_name=$db_name
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_script_progression --message="Setting up source files..." --weight=4
+
+ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 
-ynh_app_setting_set $app final_path $final_path
 # Download, check integrity, uncompress and patch the source from app.src
-wget https://github.com/facette/facette/releases/download/0.4.0rc2/facette_0.4.0rc2_jessie-amd64.deb
-dpkg -i facette_0.4.0rc2_jessie-amd64.deb
+ynh_setup_source --dest_dir=$final_path  --source_id="$architecture"
+
+chmod 750 "$final_path"
+chmod -R o-rwx "$final_path"
+chown -R $app:www-data "$final_path"
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
+ynh_script_progression --message="Configuring NGINX web server..." --weight=1
 
-# Create a dedicated nginx config
+# Create a dedicated NGINX config
 ynh_add_nginx_config
 
 #=================================================
-# CREATE DEDICATED USER
+# ADD A CONFIGURATION
 #=================================================
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-# Create a system user
-ynh_system_user_create $app
+ynh_add_config --template="../conf/facette.yaml" --destination="$final_path/facette.yaml"
 
-#=================================================
-# SPECIFIC SETUP
-#=================================================
-chown facette: /var/run/rrdcached
+chmod 400 "$final_path/facette.yaml"
+chown $app:$app "$final_path/facette.yaml"
 
 #=================================================
-# GENERIC FINALIZATION
-#=================================================
-# SECURE FILES AND DIRECTORIES
+# SETUP SYSTEMD
 #=================================================
+ynh_script_progression --message="Configuring a systemd service..." --weight=1
 
-# Set permissions to app files
-chown -R root: $final_path
+# Create a dedicated systemd config
+ynh_add_systemd_config
 
 #=================================================
 # SETUP LOGROTATE
 #=================================================
+ynh_script_progression --message="Configuring log rotation..." --weight=1
 
 # Use logrotate to manage application logfile(s)
 ynh_use_logrotate
 
 #=================================================
-# ADVERTISE SERVICE IN ADMIN PANEL
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
+
+yunohost service add $app --description="A short description of the app" --log="/var/log/$app/$app.log"
```

Comment:
```suggestion
yunohost service add $app --log="/var/log/$app/$app.log"
```

(We should either put an actual description, or no description at all, then the description from the systemd file shall be used automatically by yunohost)